docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/amazon.md

```diff
@@ -159,6 +163,317 @@ Setting the value of the tag to `owned` tells the cluster that all resources wit
+
+When creating a [custom cluster](../../../../pages-for-subheaders/use-existing-nodes.md), you must add [`--node-name`](../../reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/rancher-agent-options) to the `docker run` node registration command to set `hostname-override` -- for example, `"$(hostname -f)"`). This can be done manually or by using **Show Advanced Options** in the Rancher UI to add **Node Name**.
+
+2. Select **External (out-of-tree)** as the cloud provider. This sets `--cloud-provider=external` for Kubernetes components. This can also be done manually, by setting it in `cluster.yml`.
+
+```
+rancher_kubernetes_engine_config:
+  cloud_provider:
+    name: external
+```
+
+3. Install the AWS cloud controller manager after the cluster finishes provisioning. Note that the cluster isn't successfully provisioned and nodes are still in an `uninitialized` state until you deploy the cloud controller manager. This can be done via Helm charts in UI or manually.
+
+:::note
+
+The upstream documentation for the AWS cloud controller manager can be found [here](https://kubernetes.github.io/cloud-provider-aws).
+
+:::
+
+### Helm Chart Installation
+
+1. Click **☰**, then select the name of the cluster from the left navigation.
+
+2. Select **Apps** > **Repositories**.
+
+3. Click the **Create** button**.
+
+4. Enter `https://kubernetes.github.io/cloud-provider-aws` in the **Index URL** field.
+
+5. Select **Apps** > **Charts** from the left navigation and install **aws-cloud-controller-manager**.
+
+6. Select the namespace, `kube-system`, and enable **Customize Helm options before install**.
+
+7. Rancher-provisioned RKE nodes are tainted `node-role.kubernetes.io/controlplane` so you must update tolerations and the nodeSelector:
+
+```
+tolerations:
+  - effect: NoSchedule
+    key: node.cloudprovider.kubernetes.io/uninitialized
+    value: 'true'
+  - effect: NoSchedule
+    value: 'true'
+    key: node-role.kubernetes.io/controlplane
+
+```
+
+:::note 
+	
+There's currently [a known issue](https://github.com/rancher/dashboard/issues/9249) which doesn't allow nodeSelector to be updated from the Rancher UI.  Continue installing the chart and then edit the daemonset manually to set the `nodeSelector`:
+
+```
+nodeSelector:
+  node-role.kubernetes.io/controlplane: 'true'
+```
+
+:::
+
+8. Add `get` to `clusterRoleRules`. This allows the cloud controller manager to get service accounts upon startup.
+
+```
+  - apiGroups:
+      - ''
+    resources:
+      - serviceaccounts
+    verbs:
+      - create
+      - get
+```
+
+9. Update container arguments:
+```
+  - '--use-service-account-credentials=true'
+  - '--configure-cloud-routes=false'
+```   
+
+10. Install the chart and confirm that daemonset `aws-cloud-controller-manager` deploys successfully.
```

Comment:
can you add a note that, if using the workaround from step 7, then you should manually go and edit the daemonset at this step in order for it to deploy